exercises/G-numbers/exercise.js

```diff
@@ -1 +1,14 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+let numberOfStudents = 15;
+let numberOfMentors = 8;
+let totalStudentsAndMentors = 0;
+let message = "";
+
+// Calculate total number of students
+totalStudentsAndMentors = numberOfStudents + numberOfMentors;
```

Comment:
You can simplify this by initialising the variable and setting its value at the same time.

I.e.:

`let totalStudentsAndMentors = numberOfStudents + numberOfMentors;`

(so you no longer need line 4)